docs/data/census.md

```diff
@@ -0,0 +1,66 @@
+|Logarithm of Population Density |Logarithm of the Population Density where the Population Density is defined as the Population divided by the Land Area in square miles. |2000 Census Gazetteer Files|
+|Household Income per Capita |Aggregate household income in the 2000 census divided by the number of people aged 16-64 |2000 Census SF3 Sample Data Table P054|
+|Labor Force Participation |Share of people at least 16 years old that are in the labor force |2000 Census SF3 Sample Data Table P043|
+|Share Working in Manufacturing |Share of employed persons 16 and older working in manufacturing. |2000 Census SF3 Sample Data Table P049|
+|Fraction Foreign Born |Share of CZ residents born outside the United States |2000 Census SF3 Sample Data Table P021|
+|Fraction of Single Mothers |Number of single female households with children divided by total number of households with children |2000 Census SF3 Sample Data Table P015 |
+|Fraction of Adults Divorced |Fraction of people 15 or older who are divorced |2000 Census SF3 Sample Data Table P018|
+|Fraction of Adults Married |Fraction of people 15 or older who are married and not separated |2000 Census SF3 Sample Data Table P018|
+|Median Monthly Rent |Median "Contract Rent" (monthly) for the universe of renter-occupied housing |2000 Census SF3a (NHGIS SF3a, code: GBG) units paying cash rent |
+|Median House Price |Median value of housing units at the county level (population weighted to CZ level for CZ covariate). |2000 Census SF3a (NHGIS SF3a, code: GB7) |
+
+#### ACS5 2015 API Field Names
+
+|Name |Label |Concept |
+|:---|:---|:---|
+|[B02001_003E](http://api.census.gov/data/2015/acs5/variables/B02001_003E.json) | Black or African American alone | B02001. Race |
+|[B17020_002E](http://api.census.gov/data/2015/acs5/variables/B17020_002E.json) |Income in the past 12 months below poverty level |B17020. Poverty Status in the Past 12 Months by Age|
+|[B19025_001E](http://api.census.gov/data/2015/acs5/variables/B19025_001E.json) |Aggregate household income in the past 12 months (in 2015 Inflation-adjusted dollars) |B19025. Aggregate Household Income in the Past 12 Months (in 2015 Inflation-Adjusted Dollars) |
+|[B23025_002E](http://api.census.gov/data/2015/acs5/variables/B23025_002E.json) |In labor force |B23025. Employment Status for the Population 16 Years and Over |
+|[B16008_019E](http://api.census.gov/data/2015/acs5/variables/B16008_019E.json) |Foreign-born population |B16008. Citizenship Status by Age by Language Spoken At Home and Ability to Speak English for the Population 5+ Yrs |
+|[B09002_015E](http://api.census.gov/data/2015/acs5/variables/B09002_015E.json) |In other families:!!Female householder, no husband present |B09002. OWN CHILDREN UNDER 18 YEARS BY FAMILY TYPE AND AGE |
+
+#### Calculated Good Neighborhood Variables
+**Notes:**
+- Pseudocode includes API field names per above.
```

Comment:
Most users of this documentation won't be developers, so suggest rewording 'pseudocode'